source/fundamentals/builders/sort.txt

```diff
@@ -11,3 +11,346 @@ Sorts Builders
+   {"_id": 2, "letter": "a", "food": "donuts and coffee"}
+
+If you would like to have a guaranteed sort order 
+for documents that have fields with identical values
+you can specify additional fields to sort
+on in the event of a tie. 
+
+We can specify an ascending sort on ``letter`` followed
+by ``_id`` as follows:
+
+.. code-block:: java
+
+   import static com.mongodb.client.model.Sorts.ascending;
+
+   // <MongoCollection setup code here>
+   
+   collection.find().sort(ascending("letter", "_id"));
+
+The above code snippet returns the documents in the
+:ref:`sort_example <docs_java_fundamentals_builders_sort_sort_example>`  
+collection in the following order: 
+
+.. code-block:: json
+
+   {"_id": 2, "letter": "a", "food": "donuts and coffee"}
+   {"_id": 3, "letter": "a", "food": "maple syrup"}
+   {"_id": 5, "letter": "a", "food": "milk and cookies"}
+   {"_id": 4, "letter": "b", "food": "coffee with sugar"}
+   {"_id": 1, "letter": "c", "food": "coffee with milk"}
+   {"_id": 6, "letter": "c", "food": "maple donut"}
+
+
+Descending
+~~~~~~~~~~
+
+To specify a descending sort, use the ``Sorts.descending()``
+static factory method. A descending sort returns your documents
+from largest to smallest on the field name specified.
+
+The following code snippet shows how to specify a descending
+sort on the ``_id`` field: 
+
+.. code-block:: java
+   
+   import static com.mongodb.client.model.Sorts.descending;
+
+   // <MongoCollection setup code here>
+   
+   collection.find().sort(descending("_id"));
+
+
+The above code snippet returns the documents in the
+:ref:`sort_example <docs_java_fundamentals_builders_sort_sort_example>`  
+collection in the following order: 
+
+.. code-block:: json
+
+   {"_id": 6, "letter": "c", "food": "maple donut"}
+   {"_id": 5, "letter": "a", "food": "milk and cookies"}
+   {"_id": 4, "letter": "b", "food": "coffee with sugar"}
+   ...
+
+Note that MongoDB does not guarantee sort order for documents that have
+fields with identical values. This means that if you pass "letter" to
+the ``descending()`` method, the first document returned could be any of
+the following documents:
+
+.. code-block:: json
+
+   {"_id": 1, "letter": "c", "food": "coffee with milk"}
+   {"_id": 6, "letter": "c", "food": "maple donut"}
+
+If you would like to have a guaranteed sort order 
+for documents that have fields with identical values
+you can specify additional fields to sort
+on in the event of a tie. 
+
+The following code snippet shows how to specify a descending
+sort on the ``letter`` field followed by the ``_id`` field: 
+
+.. code-block:: java
+
+   import static com.mongodb.client.model.Sorts.descending;
+
+   // <MongoCollection setup code here>
+   
+   collection.find().sort(descending("letter", "_id"));
+
+
+The above code snippet returns the documents in the
+:ref:`sort_example <docs_java_fundamentals_builders_sort_sort_example>`  
+collection in the following order: 
```

Comment:
Nitpick: since "sort_example" isn't a keyword or reserved word, you might include "collection" which is an identifying term in the link as well.

Reply:
Thanks for the note. I don't have a strong preference either way on this note, but I will take the suggestion to see how effects the flow of the document. Makes sense that add "collection" into the link could benefit some users in how it makes the term a little more explicit. Fix: https://github.com/mongodb/docs-java/pull/59/commits/e8b1277ece7189adb784fb584ab430e7c79ec2d9